scripts/prune_lm_dir.py

```diff
@@ -328,6 +351,153 @@ def FinalizeOutput(final_work_out):
         os.remove(args.lm_dir_out + "/num_splits")
 
 
+def MatchTargetSize(num_ngrams):
+    assert(args.target_num_ngrams > 0)
+    return abs(num_ngrams - args.target_num_ngrams) / float(args.target_num_ngrams) < args.tolerance
+
+def IterateOnce(scale, step, n_iter, recovery_step, recovery_size):
+    """Prune with a specific threshold and check whether the resulting LM matches the target-num-ngrams
+
+    Args:
+        scale: the threshold equals to scale * args.threshold
+        step: number of current step
+        n_iter: number of current iteration
+        recovery_step, recovery_size: These two variables store the number of step and num-grams
```

Comment:
Remove the text 'These two variables store', which is redundant and slighlty misleading.